controller/src/main/java/io/pravega/controller/server/AuthResourceRepresentation.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server;
+
+import io.pravega.common.Exceptions;
+
+/**
+ * A utility class with methods for preparing string representations of auth-protected resources.
+ */
+public class AuthResourceRepresentation {
+
+    public static String ofScopes() {
+        return "/";
+    }
+
+    public static String ofAScope(String scopeName) {
+        Exceptions.checkNotNullOrEmpty(scopeName, "scopeName");
+        return scopeName;
+    }
+
+    public static String ofStreams(String scopeName) {
```

Comment:
Did you mean to name the argument `streams`? Are you sure it's supposed to be plural too?

Reply:
Otherwise what's the difference between this and `ofAScope`?

Reply:
Yes, I did intend the plural ``streams``. I've now added JavaDocs comments to this class and its methods now. Hopefully, the comments clarify the reason for the name and also clarify the difference between the two: ``ofStreams`` and ``ofAScope``.